plugins/chat/spec/plugin_helper.rb

```diff
@@ -80,6 +80,28 @@ def update_message(message, text: nil, user: Discourse.system_user, upload_ids:
     service_failed!(result) if result.failure?
     result.message_instance
   end
+
+  def trash_message!(message, user: Discourse.system_user)
```

Comment:
it makes tests faster and more reliable to directly call the service instead of going through all the operations in the UI, especially when making a lot of operations, like deleting X messages